Dynatrace/dynatrace-operator
# Split connection info secrets non-secret part to configmap

# Description
`TenantUUID` and `CommunicationEndpoints` and `TenantToken` were stored in single secret per ActiveGate and OneAgent. 
This PR splits first two fields to configMap leaving only `TenantToken` in secret.

`CommunicationEndpoints` are then passed to OneAgent container as env var from configMap (instead of values). Because of it, if configMap changes, OneAgent daemonset is not automatically recreated if communicationEndpoints changes.

## How can this be tested?
Run operator in cloudNativeFullStack mode and check if ActiveGate and OneAgent works properly.
Change `dynakube-oneagent-connection-info`, deamonset should not be recreated.
Restart OneAgent pod, new connectionEndpoint value should be applied:
`cat /mnt/host_root/var/lib/dynatrace/oneagent/agent/config/ruxitagentproc.conf | grep -E "tenant |serverAddress"`

## Checklist
